### ISSUE TITLE
Simplify wingets; use ahk_id to track windows 

### ISSUE THREAD

**Issue by spyoungtech** [spyoungtech/ahk#15]
Need to come up with the most conscience way to identify and track windows.

Current implementation is crude and incomplete. Would like the `Window` class to track its own `ahk_id` as a better mechanism for making sure we keep accessing the same window.

**Comment by spyoungtech** [comment]
Would be nice to have something like a selenium-like interface. e.g. methods like `find_window_by_abc` where `abc` could be any window attribute we have access to with AutoHotkey.

Additionally, maybe it would be useful to accept a callable that works like a filter.

```
def top_windows(win):
    if win.always_on_top:
        return True
ahk.find_windows(top_windows)  # find all windows that return 'True' form callable
ahk.find_window(top_windows)  #  return the first matching window
```

**Comment by spyoungtech** [comment]
resolved in #23
